src/main/java/io/harness/cf/client/dto/SegmentBuilder.java

```diff
@@ -89,10 +89,12 @@ public SegmentBuilder withVersion(Long version) {
 
   private List<Target> makeSegmentList(List<String> targets) {
```

Comment:
What do you think should we start using static analysis on Java code. e.g @Notnull

Reply:
Yes that would make sense.  It would be good to also enable some checks as part of the build